### PR TITLE
fix: fix errors from the latest dependencies

### PR DESCRIPTION
**Pull request by jk-gan**
## Summary

Ticket:

### Fixed 
- [x] Fix errors from the latest dependencies

### CODE REVIEW

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/142#" title="Author self-approved">jk-gan</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/chore/bump-version-2.0.0-alpha1/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by onimsha — APPROVED** [review]

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/142#" title="Author self-approved">jk-gan</a>*, *<a href="https://github.com/mindvalley/wukong-cli/pull/142#pullrequestreview-1612513127" title="Approved">onimsha</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/chore/bump-version-2.0.0-alpha1/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
